Create Issue templates and PR templates (#7)
Twaha-Rahman/e-commercify: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve

Reviewer: ```suggestion
about: Create a report to help us fix a bug
```

Author: Done.

title: "[BUG]"
labels: bug
assignees: ''

---

**Describe the bug**
What is working incorrectly?

***To Reproduce***

> How can we experience this error?

Reviewer: ```suggestion
How can we reproduce this bug?
```

Author: Done.

1. Go to...
2. Do...
3. See

**Expected behavior**

> What should it do?

Reviewer: ```suggestion
What do you expect it to do?
```

Author: Done.

**Screenshots**
Do you have any screenshots of the error?

**Device Information**
 - OS: [e.g. iOS]
 - Browser: [e.g. chrome, safari]
 - Version: [e.g. 22]

>  - Device Name (if mobile): [e.g. iPhone 1000]

Reviewer 2: Should we ask the user to copy/paste the output of `navigator.userAgent` ?

Author: It's an option, but then we have to parse the user agent. Also, I think it's easier for non-programming users to fill this out rather than find their user agent.
